Patches/Core/ThingDefs_Races/Races_Animal_Base.xml

```diff
@@ -40,6 +40,49 @@
 			<li Class="CombatExtended.CompProperties_TacticalManager" />																											
 		</value>
 	</Operation>
+	
+	
+	<Operation Class="PatchOperationAdd">
```

Comment:
Why patch this in, rather than just add it in Defs?

Reply:
I can't find a suitable file to put it in 😅.
Please do put it in a place that you deem suitable.